dbt/include/global_project/macros/materializations/models/incremental/incremental.sql

```diff
@@ -39,9 +39,12 @@
       {% set need_swap = true %}
   {% else %}
     {% do run_query(get_create_table_as_sql(True, temp_relation, sql)) %}
-    {% do adapter.expand_target_column_types(
-             from_relation=temp_relation,
-             to_relation=target_relation) %}
+    {% set contract_config = config.get('contract') %}
+    {% if not contract_config.enforced %}
```

Comment:
Just to confirm, since jinja can have a mind of its own, we want this behavior:

1. `contract` is in `config` and `contract.enforced == True` - don't execute
1. `contract` is in `config` and `contract.enforced == False` - execute
1. `contract` is not in `config` - execute

Is that correct? I think that's what happens here? The questionable case is 3. I'm pretty sure `{% if contract_config.enforced %}` evaluates to `false` if `contract_config` is `None`. What I don't know is if `{% if not contract_config.enforced %}` evaluates to `true` when `contract_config` is `None`.

Reply:
I've changed it to `{% if not contract_config or not contract_config.enforced %}`, which should avoid that.